src/components/childComponents/table/HuaResultsTable.jsx

```diff
@@ -0,0 +1,318 @@
+import React from "react";
+
+import {
+	Typography,
+	Table,
+	TableBody,
+	TableCell,
+	TableContainer,
+	TableHead,
+	TableRow,
+} from "@material-ui/core";
+
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import {makeStyles} from "@material-ui/core/styles";
+
+const tableColumnHead = {
+	color: "#69707F",
+	fontSize: "12px",
+	fontWeight: "500",
+	textAlign: "right",
+	border: "solid 1px rgba(46,91,255,0.08)",
+	borderRadius: "1px",
+	width: "185px",
+	backgroundColor: "rgba(234,238,255, 0.6)",
+};
+
+const tableRowHead = {
+	color: "#69707F",
+	fontSize: "12px",
+	fontWeight: "500",
+	textAlign: "center",
+	backgroundColor: "rgba(234,238,255, 0.6)",
+	borderLeft: "solid 1px #FFFFFF",
+	borderRight: "solid 1px #FFFFFF",
+	borderTop: "solid 1px #FFFFFF",
+	borderBottom: "solid 1px #FFFFFF",
+	letterSpacing: "1.13px"
+};
+
+const useStyles = makeStyles({
+	table: {
+		maxWidth: 700,
+	},
+	tableRowHeadFirst: {
+		color: "#69707F",
+		fontSize: "12px",
+		fontWeight: "500",
+		textAlign: "center",
+		backgroundColor: "rgba(234, 238, 255, 0.6)",
+		border: "solid 1px #FFFFFF",
+		letterSpacing: "1.13px"
+	},
+	tableRowHead: {
+		...tableRowHead,
+		backgroundColor: "#DB4F60",
+		color: "#FFFFFF"
+	},
+	tableRowHeadRight: {
+		...tableRowHead,
+		borderRight: "solid 1px rgba(234, 238, 255, 0.6)",
+	},
+	tableColumnHead: {
+		...tableColumnHead
+	},
+	tableColumnHeadCharacteristics: {
+		...tableColumnHead,
+		backgroundColor: "rgba(234,238,255, 1)"
+	},
+	tableColumnHeadTotal: {
+		...tableColumnHead,
+		backgroundColor: "rgba(234, 238, 255, 0.6)"
+	},
+	tableDataCell: {
+		width: "100px",
+		textAlign: "center",
+		border: "solid 1px rgba(46,91,255,0.08)"
+	},
+	tableDataCellRed: {
+		width: "100px",
+		textAlign: "center",
+		border: "solid 1px rgba(46,91,255,0.08)",
+		backgroundColor: "#FEEFF1"
+	},
+	title: {
+		height: "19px",
+		width: "460px",
+		color: "#2E384D",
+		fontSize: "16px",
+		fontWeight: "500",
+		letterSpacing: "0.6px",
+		lineHeight: "19px",
+		padding: "12px 8px"
+	},
+	radio: {
+		color: "#69707F",
+		letterSpacing: "0.53px"
+	},
+	label: {
+		fontSize: "14px"
+	}
+});
+
+
+export function HuaResultsTable(props) {
+
+	const [huaBy, setHuaBy] = React.useState('tenure');
+
+	const handleChange = (event) => {
+		setHuaBy(event.target.value);
+	};
+
+	let resultsJson = props.resultsJson;
+
+	if (resultsJson === null){
+		// to be commented out when integrated with API and prop validation can be added
+		resultsJson = {
+			"income": [
```

Comment:
I'm not sure how the "income", "race", "tenure" etc. jsons are structured in the API response. Hope this combined json structure works for integration. If they have to be separated out, we can refactor as part of separate task.

Reply:
it's separate :-(
but i can pass all three to this component and separate them inside. Don't worry :-D